content/docs/support.md

```diff
@@ -5,6 +5,6 @@ weight: 1175
 
 ## Community support
 
-If you encounter problems while using Bank-Vaults that the documentation does not address, [open an issue](https://github.com/bank-vaults/) in the repository of the relevant component, or talk to us in the [#bank-vaults channel of the Outshift Community Slack](https://outshift.slack.com/).
+If you encounter problems while using Bank-Vaults that the documentation does not address, [open an issue](https://github.com/bank-vaults/) in the repository of the relevant component, or talk to us in the [#bank-vaults channel of the CNCF Slack](https://cloud-native.slack.com/archives/C078PHYK38W/).
```

Comment:
I'd insert "you can" in front of [open an issue].

The comma is not necessary before "or talk to us..."